test/runtests.jl

```diff
@@ -101,3 +101,39 @@ let callsites = find_callsites_by_ftt(callf, Tuple{Union{typeof(sin), typeof(cos
     @test first(callsites).info isa Cthulhu.MultiCallInfo
 end
 
+let config = Cthulhu.CthulhuConfig(enable_highlighter=false)
+    for lexer in ["llvm", "asm"]
+        @test sprint() do io
+            Cthulhu.highlight(io, "INPUT", lexer, config)
+        end == "INPUT"
+    end
+end
+
+let config = Cthulhu.CthulhuConfig(
+    highlighter = `I_am_hoping_this_command_does_not_exist`,
+    enable_highlighter = true,
+)
+    for lexer in ["llvm", "asm"]
+        @test begin
+            @test_logs (:warn, r"Highlighter command .* does not exist.") begin
+                sprint() do io
+                    Cthulhu.highlight(io, "INPUT", lexer, config)
+                end == "INPUT"
+            end
+        end
+    end
+end
+
+if VERSION >= v"1.1-"
```

Comment:
This is to workaround `MethodError: no method matching rawhandle(::Base.GenericIOBuffer{Array{UInt8,1}})` in Julia 1.0 https://travis-ci.org/JuliaDebug/Cthulhu.jl/jobs/556538119#L240